### ISSUE TITLE
[Suggestion] UiModal - Option to align modal to top of viewport

### ISSUE THREAD

**Issue by nivv** [JosephusPaye/Keen-UI#287]
If the modal has changing content it is quite annoying to have the modal move all around the viewport (trying to center the modal). 

An option where you could specify to align the modal to the top of the viewport would be awesome. Basically what I'm after is to have the modal behave like the Bootstrap modal does.

![skarmklipp 2017-03-24 13 08 18](https://cloud.githubusercontent.com/assets/4033930/24293631/ff02f07c-1092-11e7-9a0b-3728ce5635d0.png)

**Comment by JosephusPaye** [comment]
Hi @nivv,
 
This is something I needed to recently and will see about adding it to the next release. Meanwhile, you can override the vertical positioning of the modal using CSS:

```scss
.ui-modal__wrapper {
    vertical-align: initial;
    padding-top: 60px; /* how far from the top */
}
```

See http://codepen.io/JosephusPaye/pen/BWgmqE for a demo.

**Comment by nivv** [comment]
@JosephusPaye Thanks! I'll do that!

**Comment by nivv** [comment]
@JosephusPaye just a heads up, the max-height for the `.ui-modal__body` needs to be re-calculated as well, otherwise the footer won't be visible.

These settings worked for me
```scss
$ui-modal-header-height         : rem-calc(136px);
$ui-modal-footer-height         : rem-calc(70px);

.ui-modal__wrapper {
    vertical-align: initial;
    padding-top: 60px; /* how far from the top */
}

.ui-modal.has-footer .ui-modal__body {
    max-height: calc(100vh - #{$ui-modal-header-height + $ui-modal-footer-height});
}
```

**Comment by JosephusPaye** [comment]
Added in https://github.com/JosephusPaye/Keen-UI/commit/0a799fb896483843c213b5dcd1a7f3731b41f0d2.

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
